packages/manager/apps/key-management-service/src/components/Listing/ListingCells.tsx

```diff
@@ -46,3 +49,62 @@ export const DatagridCellRestApiEndpoint = (props: OKMS) => {
 export const DatagridActionMenu = (props: OKMS) => {
   return <KmsActionMenu {...props} />;
 };
+
+export const DatagridServiceKeyCellName = (props: OkmsServiceKey) => {
+  const navigate = useNavigate();
+  return (
+    <div>
+      <OsdsLink
+        onClick={() => {
+          navigate(`/${props?.id}`);
+        }}
+        color={ODS_TEXT_COLOR_INTENT.primary}
+      >
+        {props?.name}
+      </OsdsLink>
```

Comment:
you can use `<Links />` to avoid setting the color. Only the name and the click action are needed